src/Avalonia.FuncUI.DSL/Primitives/TemplatedControl.fs

```diff
@@ -25,7 +25,16 @@ module TemplatedControl =
             
         static member borderBrush<'t when 't :> TemplatedControl>(color: string) : IAttr<'t> =
             Color.Parse(color) |> ImmutableSolidColorBrush |> TemplatedControl.borderBrush
+            
```

Comment:
FuncUI typically provides the following overloads for methods that take `Thickness` 

```fsharp
        static member borderThickness<'t when 't :> Border>(value: Thickness) : IAttr<'t> =
            AttrBuilder<'t>.CreateProperty<Thickness>(Border.BorderThicknessProperty, value, ValueNone)
            
        static member borderThickness<'t when 't :> Border>(value: float) : IAttr<'t> =
            value |> Thickness |> Border.borderThickness
            
        static member borderThickness<'t when 't :> Border>(horizontal: float, vertical: float) : IAttr<'t> =
            (horizontal, vertical) |> Thickness |> Border.borderThickness
            
        static member borderThickness<'t when 't :> Border>(left: float, top: float, right: float, bottom: float) : IAttr<'t> =
            (left, top, right, bottom) |> Thickness |> Border.borderThickness
```

see https://github.com/AvaloniaCommunity/Avalonia.FuncUI/blob/7edeb60c980c1850ecbddf73a2cc669ceec9f2e8/src/Avalonia.FuncUI.DSL/Border.fs#L28

Reply:
I don't think it would be worth it to add variants of all overloads that take integers.

Reply:
Fixed that.